packages/playwright-core/src/server/bidi/bidiPage.ts

```diff
@@ -530,11 +530,21 @@ export class BidiPage implements PageDelegate {
   }
 
   async setInputFiles(handle: dom.ElementHandle<HTMLInputElement>, files: types.FilePayload[]): Promise<void> {
-    throw new Error('Method not implemented.');
+    throw new Error('Setting FilePayloads is not supported in Bidi.');
```

Comment:
@yury-s, can you please explain when this is used? First I thought that you can fake files for upload by specifying a virtual file path and the file contents but it looks like (when checking some examples in the docs) it's ok to just set the file name. So what exactly would be required here from BiDi?

Reply:
Actually we can implement this one in user land, similar to juggler https://github.com/microsoft/playwright/pull/34785